cmd/sealer/cmd/cluster/run.go

```diff
@@ -98,6 +98,7 @@ func NewRunCmd() *cobra.Command {
 	runCmd.Flags().StringVarP(&runFlags.ClusterFile, "Clusterfile", "f", "", "Clusterfile path to run a Kubernetes cluster")
 	runCmd.Flags().StringVar(&runFlags.Mode, "mode", common.ApplyModeApply, "load images to the specified registry in advance")
 	runCmd.Flags().BoolVar(&runFlags.IgnoreCache, "ignore-cache", false, "whether ignore cache when distribute sealer image, default is false.")
+	runCmd.Flags().BoolVar(&runFlags.P2PDistribution, "p2p-distribution", false, "enable P2P-based image distribution, default is false.")
```

Comment:
use enum value type instead of bool type? WDYT?

Reply:
This is also a temporary solution. In the future, with the availability of other distributors, this argument will be replaced with something like "--advanced-distribution", which will adaptively select suitable distribution methods.

Reply:
ditto， maybe we can use `--distributor` flag here， and provide the following options:
1. stfp
2. p2p
3. others

Reply:
> ditto， maybe we can use `--distributor` flag here， and provide the following options:
> 
> 1. stfp
> 2. p2p
> 3. others

Sounds reasonable, updated